src/tests/JIT/Regression/JitBlue/Runtime_95315/Runtime_95315.cs

```diff
@@ -0,0 +1,56 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using Xunit;
+
+public class Runtime_95315
+{
+    [Fact]
+    public static void TestEntryPoint()
+    {
+        for (int i = 0; i < 4; i++)
+        {
+            for (int j = 0; j < 200; j++)
+            {
+                Assert.Throws<IndexOutOfRangeException>(() => Bar(new int[10], new C()));
+            }
+
+            Thread.Sleep(100);
```

Comment:
This is to allow tiering up?

Reply:
Yeah, this loop is just to tier up `Bar`. The problem is inside `Bar`